modules/core/src/main/java/org/apache/ignite/internal/processors/cache/query/continuous/CacheContinuousQueryPartitionRecovery.java

```diff
@@ -155,8 +155,8 @@ void resetTopologyCache() {
                     if (!entry.isFiltered())
                         entries.add(new CacheContinuousQueryEvent<K, V>(cache, cctx, entry));
 
-                    if (log.isDebugEnabled())
-                        log.debug("Partition was lost [lastFiredEvt=" + lastFiredEvt +
+                    if (log.isInfoEnabled())
```

Comment:
I suggest to revert this change. It's not related to the issue.

Reply:
Reverted.